### ISSUE TITLE
 Add `zalando / jackson-datatype-money`?

### ISSUE THREAD

**Issue by cowtowncoder** [FasterXML/jackson-datatypes-misc#5]
(from: https://github.com/zalando/jackson-datatype-money/issues/224)

**Comment by mathieu** [comment]
Hi @cowtowncoder,

From the original discussion from https://github.com/zalando/jackson-datatype-money/issues/224 it seems that the it’s author was willing to do just this. 

Do you really need somebody else on this integration ?

**Comment by cowtowncoder** [comment]
@mathieu Hi there! I was thinking that since actual migration itself could be relatively straight-forward task for someone with Maven knowledge, it'd be good task for Hacktoberfest participants. My intent was not bypass anyone, so I added a note on the other issue: if author or anyone else is working on this I do not want duplicate work.

Thank you for pointing this out; I realized that although github links issues via reference, it makes sense to add some not to explain addition here.

**Comment by mathieu** [comment]
Hi @cowtowncoder, it looks like I could help. Could you please explain the recommended course of actions ?

**Comment by cowtowncoder** [comment]
I assume you are familiar with creating Pull Requests part (fork this repository, make changes, create PR)?

So the idea would be to add a Maven sub-project, similar to 3 existing ones (see parent `pom.xml`, individual `subfolder/pom.xml`s).
`pom.xml` of the new sub-project needs to change; version referenced.
Should be able to copy code under `src/main/java` and `src/test/java` as is.

**Comment by mathieu** [comment]
So basically copy and paste it as is into this repo ? (with minor version and parent pom changes)

**Comment by cowtowncoder** [comment]
@mathieu correct: first step is just to make it build the same way as before, no functional changes.

Oh, and should be done for 2.12 branch.

**Comment by pgehl** [comment]
any news on this issue ?

**Comment by cowtowncoder** [comment]
@pgehl No updates here, no news. At this point blocked by https://github.com/zalando/jackson-datatype-money/issues/224 I think, so asking over there would make sense.

If anyone wants to try it -- doing PR for addition is not blocked; but we would not want to merge before Zalando folks would be ok with the move -- this would be against `2.18` branch.
As usual, I may not have time to drive this, but am happy to help guide others as needed.

**Comment by cowtowncoder** [comment]
There is now some progress, see #48!

**Comment by cowtowncoder** [comment]
Merged in `2.19` for `2.19.0`.
